### ISSUE TITLE
The X-mas - slack challenge - Part I

### ISSUE THREAD

**Issue by lakruzz** [Praqma/praqma.com#470]
_This is the first part of the X-mas challenge, which - when completed together with Part II- will eventually bring the full featured slack to Praqma - as discussed in on the [#metaslack channel](https://praqma.slack.com/files/lakruzz/F2YQ009MY/to_slack_or_not_to_slack)._

---

## Challenge: Every consultant in Praqma, shall author or co-author a blogpost

The challenge is to proof to @lakruzz that there is some ZAMM in in all the writing too, and you're more than just slackers.

The concept is that everything worth sharing should be shared - in public.

## The definition of done

- [ ] Every consultant must accept the challenge
- [ ] A post can have as many as three co-authors - The topic for the post can be chosen freely by the authors.
- [ ] Each post will have it's own separate issue in [praqma/praqma.com](https://github.com/Praqma/praqma.com/issues) (mention this issue, to make them link).
- [ ] The post shall be delivered as MarkDown in the `_posts` folder in [`praqma/praqma.com`](https://github.com/Praqma/praqma.com).
- [ ] The post shall be technically reviewed by peers before completion (a final review by @BrucePhable or @monaslarsen is considered outside this scope).
- [ ] Reviewers can not be co-authors.
- [ ] The post doesn't have to be released to the public website. In other words; it can include `published: false` in the front-matter, but it is only considered completed when it's actually delivered to the `gh-pages` branch using _the phlow_.
- [ ] The post must include at least one image (format 4:3), defined as avatar for the post.
- [ ] Even if the post contains `published: false` in the front-matter, at the time of completion, it shall be technically reviewed by peers, and a review case for either @BrucePhable or @monaslarsen shall be created.

**Comment by JKrag** [comment]
@lakruzz Could you provide or link to some sort of requirements spec for said 4:3 image?
Based on previous experience there are quite a number of implied qualifications. 

Another question: Do already published blog-posts count, or must we (co-)author a new one?

**Comment by lakruzz** [comment]
@JKrag 

Image: 
Minimum 300:225
Maximum: 2000:1500

**Comment by lakruzz** [comment]
@JKrag 

>Do already published blog-posts count...?

In your dream, honey child! 🍯  Only posts committed to `gh-pages` _after_ the date of the challenge are accepted.

**Comment by JKrag** [comment]
@lakruzz Didn't you forget the more subtle requirements like being unique for each post (i.e. not used before), not stolen from somewhere else and probably not drawn by someone with my approximate drawing talents?

**Comment by lakruzz** [comment]
@JKrag 

The challenge actually has a purpose: Slack can not ever be a substitute for **DORITH**. When _everyone_ has made them selve acquainted with the process of releasing a **ZAMM** quality blog posts on our web site, then I know, that at least it's not the process that's the impediment.

There are two important principles that needs to be present in the releases:

- DORITH
- ZAMM

## DORITH
**DO**ing the **RI**ght **TH**ing. as in Don't put things that really should be a blog post in Slack - in fact I double dare you, find something that is already in Slack, but really should have been published - and publish it.

## ZAMM
ZAMM is short for “ZEN and the Art of Motorcycle Maintenance”. A book I read 30 years ago, and then again this summer. It’s not at all about motorcycles. It’s about a guy who travels on motorcycle with his son from Minneapolis to San Francisco over mountains and through prairie and desert. Soon the reader realizes that this guy has a dark past, He has recently recovered from a previous life as professor in rhetorik and philosophy. 

His research ended up driving him insane, literally. 

He was institutionalized and he is recovered, because he was treated with hundreds of electroshock treatments.

Now, on this journey, with his son on the back of a motorcycle, he is trying to get a glimpse back into his former self, that was reset and wiped out by the electroshock treatments, but still was on to something which he vaguely recall as important.

Back then, as a professor, he was trying to define Quality - _The quest for the definition of quality, was what drove him insane._ 

ZAMM has the following morale; quality is not a tangible thing, it may not even be of this world, it may very well be a pre-existence force, that is the catalyst for subject and matter to interact and become the world as we know it (pretty wild - read it!)

In short, quality can not be described BUT the then again I bet:

* You’ll know it when you see it! 
* and you’ll know when it’s missing!

Let's use the ZAMM definition of quality for these blog posts - I have made the requirements far more precise - I've added the following:

- [ ] The post shall have a sprinkle of both DORITH and ZAMM
